.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -41,10 +42,16 @@
   <!-- Update the Lightshow URLs below to show the new and old grammars in action. -->
   - Old: https://github-lightshow.herokuapp.com/
   - New: https://github-lightshow.herokuapp.com/
-  
+
 - [ ] **I am updating a grammar submodule**
   <!-- That's not necessary, grammar submodules are updated automatically with each new release. -->
 
 - [ ] **I am adding new or changing current functionality**
   <!-- This includes modifying the vendor, documentation, and generated lists. -->
   - [ ] I have added or updated the tests for the new or changed functionality.
+
+- [ ] **I am changing the color of a language**
```

Comment:
I'd use

> I am changing the colour **_associated with a language_**

Since language is a human concept with no baryonic form capable of absorbing or reflecting bands of the visible electromagnetic spectrum. =)